packages/server/__snapshots__/3_plugins_spec.js

```diff
@@ -402,3 +343,121 @@ An invalid configuration value returned from the plugins file: \`cypress/plugins
 Expected \`viewportWidth\` to be a number. Instead the value was: \`"foo"\`
 
 `
+
+exports['e2e plugins fails when there is an async error at the root 1'] = `
+
+The following error was thrown by a plugin. We stopped running your tests because a plugin crashed. Please check your plugins file (\`/foo/bar/.projects/plugins-root-async-error/cypress/plugins/index.js\`)
```

Comment:
this error says "We stopped running your tests...", but then a few lines later, we say "Run Starting" - i think this could be confusing. maybe we should always print the "Run Starting" heading first.

Reply:
That's a good point. The timing with these things is always tricky since it's an error that occurs asynchronously and could occur at any given time, but I agree, we shouldn't say the run is starting if it's already failed. Probably should be able to prevent 'Run Starting' from printing altogether in that case.

Reply:
That sounds good, either preventing it from starting, or putting it before this error message